src/api/app/assets/stylesheets/webui/datatables.scss

```diff
@@ -33,3 +33,9 @@ table.dataTable.table-sm {
 .table-hover tbody tr:hover {
   background-color: $gray-300;
 }
+
+table.dataTable {
+  .quality-category.badge {
+    margin-left: 0.5em;
```

Comment:
You can also use the [bootstrap spacing](https://getbootstrap.com/docs/4.3/utilities/spacing/#examples) classes to do simple things like this.

Reply:
Ok